src/style.css

```diff
@@ -1 +1,79 @@
 /* Styles go here */
+/* stylelint-disable-next-line string-quotes */
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap');
+
+body,
+ul {
+  all: initial;
+  box-sizing: border-box;
```

Comment:
move to `*` selector

Reply:
okay, but on checklist was said don't use `*` selector for zeroing out your margins or paddings